lib/mastodon/cli/media.rb

```diff
@@ -317,6 +317,22 @@ def lookup(url)
 
     private
 
+    def media_attachment_storage_size
+      MediaAttachment.sum(file_and_thumbnail_size_sql)
+    end
+
+    def local_media_attachment_storage_size
+      MediaAttachment.where(account: Account.local).sum(file_and_thumbnail_size_sql)
+    end
+
+    def file_and_thumbnail_size_sql
+      Arel.sql(
+        <<~SQL.squish
```

Comment:
Would this be the same as a single quoted string inside the brackets, or is it running into a line length issue with that format?

Reply:
If you mean "what does ruby do with this?" - yes, it's basically the same as a single quoted non-interpolated string.

If you mean "why is it like this, then?!" - the differences are just:

- The `Arel.sql` wrapper marks the string as safe to use/compose into larger sql statements (not super relevant in this CLI usage, but general best practice on raw sql strings)
- The `SQL.squish...` heredoc is a nicety for editors -- most modern editors will use language appropriate syntax highlighting (sql in this case) inside the heredoc block when tagged this way